src/registrar/admin.py

```diff
@@ -222,11 +256,97 @@ class ContactAdmin(ListHeaderAdmin):
 
     search_fields = ["email", "first_name", "last_name"]
     search_help_text = "Search by firstname, lastname or email."
+    list_display = [
+        "contact",
+        "email",
+    ]
+
+    # We name the custom prop 'contact' because linter
+    # is not allowing a short_description attr on it
+    # This gets around the linter limitation, for now.
+    def contact(self, obj):
+        if obj.first_name or obj.last_name:
+            return obj.get_formatted_name()
+        elif obj.email:
+            return obj.email
+        elif obj.pk:
+            return str(obj.pk)
+        else:
+            return ""
```

Comment:
```suggestion
            return None
```
(Optional) Error handling/functionality wise, there is a possibility for things to get a bit buggy when you return empty strings as opposed to None. I'd say this is fine to keep if you need this down the line (i.e. using it in an HTML file), but returning None is a bit more predictable imo if we're using that

Reply:
I want to return something here otherwise the list table may break. I'll test what None does. If the table does not break, I'll accept your change. If the table breaks, I'd rather stick with an empty string.

Reply:
@rachidatecs If the table expects an empty string I'd say it makes the most sense to keep that - agreed. Not worth it if it breaks things